tv/interfaces/session/index.ts

```diff
@@ -0,0 +1,54 @@
+export interface RoomResult {
```

Comment:
It feels weird to have room-related interfaces in a session file. Could we use the `Result` and `AllDemoRoom` interfaces from the rooms file instead?

Reply:
Fixed